### PR TITLE
Support for Finnish language and tiny size setting

### PR DESCRIPTION
**Pull request by jannekalliola**
This pull request adds support for Finnish language and also a smaller than small size ("tiny"), as Finnish words are quite long and the clock would not fit to my screen even on small setting.

### CODE REVIEW

File: css/MMM-text-clock.css [hunk 1]
```diff
@@ -22,6 +22,14 @@
   display: grid;
 }
 
+.MMM-text-clock div.module-content div.tiny {
+  font-size: 15px;
```
Comment (ngnijland) [inline css/MMM-text-clock.css:26]: I'm using MagicMirror's [css classes](https://github.com/MichMich/MagicMirror/blob/master/css/main.css#L62) to set the font size of the clock based on the `size` setting in the config. Although I like `tiny` as a word and config better I think it's easier if it is renamed to `xsmall`. That way this piece of CSS can be removed and font-size will be set automatically.

The `xsmall` class sets the font-size to `1rem` which will convert to 16px in most cases. I see you put the font-size to 15px. Is 16px okay as wel? 

If not I would suggest the following:
Leave the size named `tiny`, change the selector to `.grid--tiny` and add the following to `MMM-text-clock.js`  at line ~224:
```
if(this.size === 'tiny') {
  grid.classList.add('grid--tiny');
}
```

Hope this is clear :)

**Review by ngnijland — CHANGES_REQUESTED** [review]
Hi @jannekalliola! Thanks for taking the time to create a pull request to this module! Overall the changes look rock solid 🙏 

I did add one comment on the implementation of the font-size for the `tiny` size config. Let me know if you have any questions.

File: css/MMM-text-clock.css (hunk 1 above)
Comment (jannekalliola) [reply css/MMM-text-clock.css:26]: Thanks for a quick response. I didn't know that xsmall exists, I'll convert my changes to use it.

Comment (ngnijland) [reply css/MMM-text-clock.css]: Thanks!

**Comment by ngnijland** [comment]
@jannekalliola the pipelines fail on two linter errors. If fix those I'll merge the PR!

**Review by ngnijland — APPROVED** [review]

**Pull request merged**